bin/flutterflow_cli.dart

```diff
@@ -44,17 +44,30 @@ void main(List<String> args) async {
   }
 
   try {
-    await exportCode(
-      token: token,
-      endpoint: endpoint,
-      projectId: project,
-      destinationPath: parsedArguments.command!['dest'],
-      includeAssets: parsedArguments.command!['include-assets'],
-      branchName: parsedArguments.command!['branch-name'],
-      unzipToParentFolder: parsedArguments.command!['parent-folder'],
-      fix: parsedArguments.command!['fix'],
-      exportAsModule: parsedArguments.command!['as-module'],
-    );
+    switch (parsedArguments.command?.name) {
+      case 'export-code':
+        await exportCode(
+          token: token,
+          endpoint: endpoint,
+          projectId: project,
+          destinationPath: parsedArguments.command!['dest'],
+          includeAssets: parsedArguments.command!['include-assets'],
+          branchName: parsedArguments.command!['branch-name'],
+          unzipToParentFolder: parsedArguments.command!['parent-folder'],
+          fix: parsedArguments.command!['fix'],
+          exportAsModule: parsedArguments.command!['as-module'],
+        );
+        break;
+      case 'firebase-deploy':
+        await firebaseDeploy(
```

Comment:
I'm getting several errors when running this command:

1) `Error: Unable to load .../custom_cloud_functions/package.json: SyntaxError: No data, empty input at 1:1`
2) Error: Directory 'public' for Hosting does not exist.
3) Error: Cannot release resource type "firebase.storage"

My test project: https://app.flutterflow.io/project/misc-firebase2-72cu1x

Reply:
Should be fixed now